components/productions/index/events.vue

```diff
@@ -2,13 +2,13 @@
   <section>
     <template v-if="!isLoading">
       <div class="heading-container">
-        <h2>
+        <pub-title class="page-title">
```

Comment:
Nit: The `page-title` class name feels a bit confusing because `pub-page-title` is also a component.

Reply:
It's also not really a page title I think?

Reply:
I mean, they're all titles. The ones I replaced are all h2's. But if you have a better name we can still change it.

Reply:
I meant it's not really a _page_ title 😉 (I was talking about the class name, not the component name) 

I think it's especially confusing with the `pub-page-title` component. Because what would be the difference between `<pub-title class="page-title">` and `<pub-page-title>` for someone going through the code to make a change?

Reply:
Since you already have `.heading-container`, I guess `.heading` would be okay?

Reply:
ahhh ok, I totally missed your point! Yeah sure I'll change them.

Reply:
done